README.md

```diff
@@ -165,6 +165,10 @@ Gets the list of pages in a given namespace - [read more](http://www.mediawiki.o
 
 Gets the list of pages by a given prefix - [read more](https://www.mediawiki.org/wiki/API:Allpages)
 
+### bot.getPagesTranscludedBy(page, callback)
+
+Gets the list of pages that are transcluded by the given page - [read more](https://www.mediawiki.org/wiki/API:Transcludedin)
```

Comment:
That's right, thank you for checking that and making it clearer. I just corrected it. :smile: